RssReader/rss_parser.py

```diff
@@ -0,0 +1,108 @@
+"""
+This module interchange between Internet and program
+"""
+
+import RssReader.feedparser as feedparser
+import RssReader.html2text as html2text
+import re
+import datetime
+import time
+import urllib.request
+
+
+def get_rss(url: str):
+    """This function receives the answer from server"""
+
+    news = feedparser.parse(url)
+    if news.entries:
```

Comment:
`return news if news.entries else None`

Reply:
The line was found and fixed. It will be updated during the next "push" iteration.